In [2]:
name = "Emily"
hourly_wage = 35.00
hours = 40
weekly_pay=hourly_wage*hours
print(f"{name} earns {weekly_pay} per week")

Emily earns 1400.0 per week


In [12]:
temps = [61, 65, 72, 58, 70, 74, 68]
days_over_65=0
for temp in temps:
    if temp >65:
        days_over_65=days_over_65 +1
print(days_over_65)

4


In [13]:
def to_celsius(f):
    return(f-32)*5/9
print(round(to_celsius(temps[0]),1))

16.1


In [15]:
movie={"title":"Dune","year":2021,"rating":8.0}
for key,value in movie.items():
    print(key,":",value)

title : Dune
year : 2021
rating : 8.0


In [24]:
import pandas as pd
df = pd.read_csv("chickwts.csv",index_col=0)

AI use: was getting information for 3 columns because the numbers column in the data was being counted as "Unnamed". I used AI how to ask it to adjust this, it said to have it treated as an index column =0 so that it would only calculate weight and feed. I then re-ran each cell to remove the column.

In [25]:
df.head

<bound method NDFrame.head of     weight       feed
1      179  horsebean
2      160  horsebean
3      136  horsebean
4      227  horsebean
5      217  horsebean
..     ...        ...
67     359     casein
68     216     casein
69     222     casein
70     283     casein
71     332     casein

[71 rows x 2 columns]>

In [26]:
df.shape

(71, 2)

In [27]:
df.describe()

,weight
count,71.000000
mean,261.309859
std,78.073700
min,108.000000
25%,204.500000
50%,258.000000
75%,323.500000
max,423.000000


The describe output tells me that the average weight is 261.3 for the 71 chicks. Describe output also tells me the percentiles which I find interesting to analyze the data. From my understanding, 75% of the chicks weigh 323.5 or less, which also means that only about 25% weigh 323.5 or more. It is interesting to see how the chick weights are distributed by the percentiles and how the heaviest chicks are significantly heavier than the average of 261.3.